### DISCUSSION TITLE
Release 5.0.1 Sunbird ED Regression Status

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#202] [General Discussion]
Hi All,

Please find the testing status of R5.0.1 as of 29th Sep 2022

- Executing NF test cases for [SB-30516](https://project-sunbird.atlassian.net/browse/SB-30516) For Avanti

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (52.6% Completed)
Planned Sign-Off Date: Oct 6, 2022

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Execution Status: 

![image](https://user-images.githubusercontent.com/40622169/193062943-21a34090-16c2-41a2-9769-96e395d5d8e8.png)

![image](https://user-images.githubusercontent.com/40622169/193063082-7256a3f9-40db-4db5-aa6f-2a93c4959cf0.png)

Defect Status:

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Defect ID | Summary | Status | Assignee
-- | -- | -- | --
ED-244 | Mobile App: "Download" text in button is displayed in 2 lines in the Certificate | Assigned | Sujith
ED-246 | Mobile App: Getting Failed message when trying to add Recovery ID and Mobile number in profile page | Open | Sujith
ED-291 | In My groups page: Changing to any regional language is not working as Expected. | Open | Rajeev
SB-31017 | Unable to update / edit the Not accepted interactive Question set in the cokreat contribution portal. | Open | Kshitija
SB-31019 | Mobile App (4.9 Build): Content details page is retained when the question set started to play in the interactive video in portrait mode | Open | Kshitija
SB-31020 | Mobile App (4.9 Build): Interactive video resource alone or added in Textbook or Course, the maximize button is not working only playing in portrait mode | Open | Kshitija
SB-31021 | Mobile App (4.9 Build): When the interactive video for the first time in Course/ Textbook the interactive question is not displayed or not played, Second time if we open the same content it is displayed. | Open | Kshitija
SB-31022 | After consuming the course partially or Completely , Course progress is not updating . | Open | Kshitija

</body>
</html>

CC @sharathkashyap @rajeevsathish @luckynvdu @karn-nahata @chitranshu-keshav @ShailandraOjha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.1 as of 30th Sep 2022

-Executing NF test cases for [SB-30516](https://project-sunbird.atlassian.net/browse/SB-30516) For Avanti
- We have 60 Failed/Blocked test cases for the defect id's SB-31041,SB-31035,SB-31019,SB-31020,SB-31046 assigned to Avanti team.

**Environment:** Sunbird ED Staging
**Testing completion percentage:** In Progress (64.89% Completed)
**Planned Sign-Off Date:** Oct 6, 2022

**Execution sheet:** [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

**Execution Status:**
![image](https://user-images.githubusercontent.com/40622169/193284560-f1ef447f-7ee2-4dc5-b99d-f4c897de528a.png)

![image](https://user-images.githubusercontent.com/40622169/193284745-04fee2f6-0aca-4f07-82f3-9184e91c05c1.png)

**Defect Status:**

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Defect ID | Summary | Status | Assignee
-- | -- | -- | --
ED-244 | Mobile App: "Download" text in button is displayed in 2 lines in the Certificate | Assigned | Sujith
ED-246 | Mobile App: Getting Failed message when trying to add Recovery ID and Mobile number in profile page | Open | Sujith
ED-291 | In My groups page: Changing to any regional language is not working as Expected. | Open | Rajeev
SB-31017 | Unable to update / edit the Not accepted interactive Question set in the cokreat contribution portal. | Open | Kshitija
SB-31019 | Mobile App (4.9 Build): Content details page is retained when the question set started to play in the interactive video in portrait mode | Open | Kshitija
SB-31020 | Mobile App (4.9 Build): Interactive video resource alone or added in Textbook or Course, the maximize button is not working only playing in portrait mode | Open | Kshitija
SB-31021 | Mobile App (4.9 Build): When the interactive video for the first time in Course/ Textbook the interactive question is not displayed or not played, Second time if we open the same content it is displayed. | Open | Kshitija
SB-31041 | Mobile app (4.10): If user opens the interactive video player in any trackable or non trackable collection/book then it is directly opening up in fullscreen mode and not able to change back to portrait mode | Open | Kshitija
SB-31040 | Mobile App (4.9 Build): When interactive content is played in the non trackable Collection, the interactive Question Set is not displayed only the video is played | Open | Kshitija
SB-31035 | Mobile app (4.10): If user opens the interactive video player which goes to the fullscreen mode and reaches the interception point, fullscreen mode is not exited and question set is loaded in the full screen mode only which is not as expected | Open | Kshitija


</body>
</html>

CC @sharathkashyap @rajeevsathish @luckynvdu @karn-nahata @chitranshu-keshav @ShailandraOjha

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.1 as of 3rd Oct 2022


We have 54 Failed/Blocked test cases for the defect id's SB-31041,SB-31019,SB-31020,SB-31046 assigned to Avanti team.

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (83.59% Completed)
Planned Sign-Off Date: Oct 6, 2022

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/193580347-43461bb0-bd96-4948-9025-d248aef28800.png)

![image](https://user-images.githubusercontent.com/40622169/193580440-4cf400c2-6c9a-4a37-9730-d7ade38a49a6.png)

Defect Status:

<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Issue Id | Issue description | Status | Asignee
-- | -- | -- | --
SB-31017 | Unable to update / edit the Not accepted interactive Question set in the cokreat contribution portal. | Open | Kshitija
SB-31019 | Mobile App (4.9 Build): Content details page is retained when the question set started to play in the interactive video in portrait mode | Open | Kshitija
SB-31020 | Mobile App (4.9 Build): Interactive video resource alone or added in Textbook or Course, the maximize button is not working only playing in portrait mode | Open | Kshitija
SB-31021 | Mobile App (4.9 Build): When the interactive video for the first time in Course/ Textbook the interactive question is not displayed or not played, Second time if we open the same content it is displayed. | Open | Kshitija
SB-31041 | Mobile app (4.10): If user opens the interactive video player in any trackable or non trackable collection/book then it is directly opening up in fullscreen mode and not able to change back to portrait mode | Open | Kshitija
SB-31040 | Mobile App (4.9 Build): When interactive content is played in the non trackable Collection, the interactive Question Set is not displayed only the video is played | Open | Kshitija
SB-31046 | Mobile App (4.9 Build): If user opens the interactive video player in any trackable collection/ book it is directly opening up in landscape mode and reaches the interception point, then question set is displayed but froze and progress is not updated | Open | Kshitija
ED-244 | Mobile App: "Download" text in button is displayed in 2 lines in the Certificate | Assigned |Sujith
ED-246 | Mobile App: Getting Failed message when trying to add Recovery ID and Mobile number in profile page |Open|Sujith

</body>
</html>

CC @sharathkashyap @rajeevsathish @luckynvdu @karn-nahata @chitranshu-keshav @ShailandraOjha

Regards,
Anusha

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.1 as of 6th Oct 2022

**-We have completed the execution and waiting for defect fixes.
-We have 3 Open defects LR-230,LR-231,SB-31065 for which we do not have any ETA**.


Environment: Sunbird ED Staging
Testing completion percentage: In Progress (98.57% Completed)
Planned Sign-Off Date: Oct 6, 2022

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/194322124-6149c108-8e0c-4a3c-b01b-cef3b89aa56c.png)

![image](https://user-images.githubusercontent.com/40622169/194322330-98135235-6453-4b3b-a400-adfad676d70b.png)

Defect Status:

<html>
<body>
<google-sheets-html-origin>

Issue Id | Issue description | Status | Asignee
-- | -- | -- | --
LR-230 | [Admin> Manage > Consent user details] Consent user report data is not getting generated in csv file | Open | Hari
LR-231 | All exhaust reports submitted in course dashboard after running job is showing as Failed status | Open | Reshmi
SB-31065 | Mobile app: If user enrolls into any course, trackable book/collection and taps on Start learning/Continue learning then, video player is not getting opened and player is not visible on the screen | Open | Kshitija


</body>
</html>

CC @sharathkashyap @rajeevsathish @luckynvdu @karn-nahata @chitranshu-keshav @ShailandraOjha @reshmi-nair

Regards,
Anusha

**Reply by ShailandraOjha** [reply]
@reshmi-nair and @karn-nahata Need your help to get the fixes by tomorrow's first half to sign off the release. Today was the planned date for sign off.

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.0.1 as of 7th Oct 2022

-We have completed the execution and waiting for defect fixes.
-We have 1 Open defect SB-31065 for which we have got ETA as today EOD.

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (99.64% Completed)
Planned Sign-Off Date: Oct 6, 2022

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1pP1eRNXYRNTiJe0BYZ2BowHRJ9ygCi6VKIRSJ2cgFMM/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/194562280-3652765c-c209-4545-b7ca-c140d448bba5.png)

![image](https://user-images.githubusercontent.com/40622169/194562407-d44c8b46-0b94-4af6-8665-488e2defcc5e.png)

Defect Status:
<html>
<body>
<google-sheets-html-origin>

Issue Id | Issue description | Status | Asignee
-- | -- | -- | --
SB-31065 | Mobile app: If user enrolls into any course, trackable book/collection and taps on Start learning/Continue learning then, video player is not getting opened and player is not visible on the screen | Open | Kshitija


</body>
</html>

CC @sharathkashyap @luckynvdu @karn-nahata @chitranshu-keshav @ShailandraOjha 

Regards,
Anusha
